### ISSUE TITLE
Check boxes in export options window unlabeled to screen readers

### ISSUE THREAD

**Issue by TheQuinbox** [NationalSecurityAgency/ghidra#6279]
**Describe the bug**
The check boxes in the export options window aren't labeled. For example, if I select C/C++ and then hit options, my screen reader just says "check box unchecked" for the check box to create a header file. I can use screen review and get the label of the control that way, so the labels do exist, they're just not properly attached to the controls.

**To Reproduce**
1. Load a project in Ghidra, with NVDA running.
2. Open an executable in the code browser.
3. Choose file -> Export
4. Press the options button and tab around.

**Expected behavior**
The already existing labels should be attached to the check boxes, so I can simply tab and know what I'm focused on.

**Environment (please complete the following information):**
 - OS: Windows 10 22H2
 - Java Version: openjdk 11.0.16.1 2022-08-12 LTS
 - Ghidra Version: 11.0.1-20240130
 - Ghidra Origin: Scoop

**Comment by ghidragon** [comment]
I've add that to my list. There are many components that don't describe themselves very well and I'm fixing them as I find them...
